core/types/src/persistence.ts

```diff
@@ -1,38 +1,36 @@
-export type Commitment = {
-  // currentHash = H(Underlying data/doc + salt)
-  salt: string;
-  // Values needed for verification (Shield.verifyAndPush inputs)
-  value: string;
-  proof: number[];
-  publicInputs: string[];
-  signatures: object; // Allows mapping from participant address -> signature
-  metadata: object;
-  sender: string; // Address of participant who created the commitment
-};
+import { Circuit, Commitment } from "./privacy";
 
 export type Participant = {
-  address: string; // Can use this to lookup messagingEndpoint, etc.
-  metadata: object;
+  address: string; // public address of the participant
```

Comment:
Which address is this? Ethereum public key?

Reply:
Yes.